dockerfiles/archlinuxarm/Dockerfile.archlinuxarm

```diff
@@ -20,19 +20,10 @@ RUN pacman --noconfirm -Syu --needed \
     && pacman --noconfirm -Rs \
         base \
         dhcpcd \
-        jfsutils \
-        lvm2 \
-        man-db \
-        man-pages \
```

Comment:
This is weirdly weird. I don't think that man-db could just disappear - wasn't it renamed?